docs/run-tests-on-local.md

```diff
@@ -79,8 +79,44 @@ Open the report file by browser: ${WORK_DIR}/TAF/testArtifacts/reports/cp-edgex/
     ```
 3. Run Test
     ###### Run V2 API Functional testing:
+    
+    - For Core Command: Not supported. The script will auto deploy device-virtual through compose file.
```

Comment:
Could you modify the test script so that we can manually deploy the device-virtaul and run the testing?

Reply:
It will affect lots of tests, I prefer to update it together with issue #315.

Reply:
No problem, I'm fine with that.